content/en/core/overview/architecture.md

```diff
@@ -32,13 +32,70 @@ A free and open source SQL database that we use for analytics queries for displa
 
 ### cht-core
 
-[cht-core](https://github.com/medic/cht-core) is the application that most users interact with. It's an [AngularJS](https://angularjs.org) single page responsive web application.
+[cht-core](https://github.com/medic/cht-core) is composed of the following web applications:
 
-We use an offline first strategy which means the data is stored on the client and all pages can load immediately regardless of whether you have a fast connection, slow connection, or no connection at all. The data is stored in [PouchDB](https://pouchdb.com) which replicates changes back and forth in the background with the server CouchDB.
+#### CHT Web Application
```

Comment:
I struggled a bit with webapp name, so instead of "webapp" I used "CHT Web Application" because I noticed that "admin" is called "App Management" in some other places. Is it okay?